layouts/index.html

```diff
@@ -54,8 +54,8 @@ <h1 class="bold-title {{ if or (.Site.Params.fadeIn | default true) .Site.Params
         {{ end }}
 
         {{ range sort $pages "Params.weight" }}
-        {{ if ne .Name "contact.md" }}
         {{ if .File }}
+        {{ if or (ne (os.Stat .File.Dir).Name "home") (ne .File.TranslationBaseName "contact") }}
```

Comment:
Why do you need to check both `(os.Stat .File.Dir).Name` and `.File.TranslationBaseName `? Isn't `File.TranslationBaseName` enough?

Reply:
It is just in case someone had a "contact" project or a "contact" blog post.

Reply:
How does this fix this? Can you please elaborate a bit?

Reply:
hummm. I guess that you are right and checking the directory is not needed. I will update the PR accordingly.

Reply:
But I think you need to check `.IsPage`.

Reply:
I am not sure on how to use the `.isPage` check. Do I need to check `.IsPage` when I skip the contact page, when I filter to get the contact only or in both cases?